test/built-ins/Temporal/Calendar/prototype/yearMonthFromFields/overflow-undefined.js

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2021 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.calendar.prototype.yearmonthfromfields
+description: Temporal.Calendar.prototype.yearMonthFromFields will return correctly with default overflow which constrain date range.
```

Comment:
Consider combining this test with https://github.com/tc39/proposal-temporal/blob/main/polyfill/test/Calendar/prototype/yearMonthFromFields/overflow-undefined.js